common/src/main/java/com/wynntils/features/user/redirects/TerritoryMessageRedirectFeature.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user.redirects;
+
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.notifications.NotificationManager;
+import com.wynntils.mc.event.SubtitleSetTextEvent;
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.wynn.event.ChatMessageReceivedEvent;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class TerritoryMessageRedirectFeature extends UserFeature {
+    private static final Pattern TERRITORY_MESSAGE_PATTERN = Pattern.compile("§7\\[You are now (\\S+) (.+)\\]");
+
+    @Config
+    public boolean redirectTerritoryMessages = true;
+
+    // Handles the subtitle text event.
+    @SubscribeEvent
+    public void onSubtitleSetText(SubtitleSetTextEvent event) {
+        if (!redirectTerritoryMessages) {
+            return;
+        }
+        String codedString = ComponentUtils.getCoded(event.getComponent());
+        Matcher matcher = TERRITORY_MESSAGE_PATTERN.matcher(codedString);
+        if (matcher.matches()) event.setCanceled(true);
+        String direction = matcher.group(1);
+        String rawTerritoryName = matcher.group(2);
+        // Want to account for weird stuff like "the Forgery" and make it "The Forgery" for the sake of our brief
+        // message (looks odd otherwise).
+        String territoryName = rawTerritoryName.substring(0, 1).toUpperCase() + rawTerritoryName.substring(1);
+
+        String enteringMessage = String.format("§7Now §o%s§r %s", direction, territoryName);
```

Comment:
I think we could shorten this with using an arrow emoji, like friend join message does.

Reply:
See PR Body for new example with emoji. I think the italicization still "works", but let me know if you want me to axe it.

Reply:
I would just straight up remove the entering/leaving part. No need for it. Arrows are condense and easy to understand. I would however, make the whole message gray, territory name included. 

As much as you like your screen/chat not the be spammed with random stuff, I like it when notifications are short and simple to understand.

Reply:
Updated, will show up in coming commits.

Reply:
PR Body Image updated